File: VRP-Core/src/main/java/VRPCore/Commands/WantedCommand.java
package VRPCore.Commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

import VRPCore.VRPCore;
import VRPCore.Models.Player;

public class WantedCommand implements CommandExecutor {
	
	private VRPCore core;
	
	public WantedCommand(VRPCore core) {
		this.core = core;
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		org.bukkit.entity.Player player = (org.bukkit.entity.Player)sender;
		
		if(args.length >= 1) {
			if(args[0].equalsIgnoreCase("set")) {
				if(args.length >= 2){
					Player _player = core.playerManager.GetPlayer(player.getUniqueId());
					_player.setWantedLevel(Float.parseFloat(args[1]));
				}
			}
		}
		
		return true;
	}
}

File: VRP-Core/VRP-Core/src/main/kotlin/Commands/WantedCommand.kt
package VRPCore.Commands

import VRPCore.Models.VRPlayer
import VRPCore.VRPCore
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class WantedCommand(private val core: VRPCore) : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val player = sender as Player
        if (args.isNotEmpty()) {
            if (args[0].equals("set", ignoreCase = true)) {
                if (args.size >= 2) {
                    val _player: VRPlayer = core.playerManager.GetPlayer(player.uniqueId)
                    _player.setWantedLevel(args[1].toFloat())
                }
            }
        }
        return true
    }
}